### PR TITLE
index.html: Remove unneeded end tag.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>{{title}}</title>
-    <link rel="canonical" href="https://www.srihash.org"/>
+    <link rel="canonical" href="https://www.srihash.org/">
```
Comment (XhmikosR) [inline templates/index.html]: BTW the idea was to actually do this dynamically and not hardcoded like now. Not sure how to get the value from Hapi though.

Comment (mozfreddyb) [reply templates/index.html]: Dynamically, why? I thought the idea was to say that one domain should be preferred over the other and that's it.

Comment (XhmikosR) [reply templates/index.html]: Dynamically meaning, get the current domain Hapi is using and not hardcoding it like this.

Comment (XhmikosR) [reply templates/index.html]: But anyway, this isn't something to deal with on this PR.

**Pull request merged**